code/datums/components/deadchat_control.dm

```diff
@@ -0,0 +1,243 @@
+///Will execute a single command after the cooldown based on player votes.
+#define DEMOCRACY_MODE (1<<0)
```

Comment:
Undef these please, or if you need them in other files, prefix them with ghostcontrol or something.

Reply:
Addressed, moved em to out to a defines file as well